lib/modules/manager/cargo/update-locked.ts

```diff
@@ -0,0 +1,23 @@
+import { logger } from '../../../logger';
+import type { UpdateLockedConfig, UpdateLockedResult } from '../types';
+import { extractLockFileContentVersions } from './locked-version';
+
+export function updateLockedDependency(
+  config: UpdateLockedConfig
+): UpdateLockedResult {
+  const { depName, currentVersion, newVersion, lockFile, lockFileContent } =
+    config;
+  logger.debug(
+    `cargo.updateLockedDependency: ${depName}@${currentVersion} -> ${newVersion} [${lockFile}]`
+  );
+  try {
+    const locked = extractLockFileContentVersions(lockFileContent ?? '');
```

Comment:
```suggestion
    const locked = extractLockFileContentVersions(lockFileContent!);
```
should never null or undefined here.